easybuild/easyconfigs/g/gffread/gffread-0.12.7-GCC-11.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'MakeCp'
+
+name = 'gffread'
+version = '0.12.7'
+
+homepage = 'https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread'
+description = """GFF/GTF parsing utility providing format conversions,
+region filtering, FASTA sequence extraction and more."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
```

Comment:
since this is a parsing utility, is there a reason to think that using different compilers will bring a significant performance benefit? 

(if not we typically use `GCCcore` because then the same easyconfig/module can then be used as a dependency for other toolchains such as 'intel')

Reply:
Thanks for the hint and the explanation, I will change this.